builds/misc/images-release.yaml

```diff
@@ -0,0 +1,601 @@
+              ]
+          signConfigType: inlineSignParams
+      - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+        displayName: "Functions Binding Code Sign"
+        inputs: 
+          ConnectedServiceName: "Azure IoT Edge Code Sign 2"
+          FolderPath: $(Build.BinariesDirectory)/publish/Microsoft.Azure.WebJobs.Extensions.EdgeHub
+          Pattern: Microsoft.Azure.WebJobs.Extensions*.dll
+          SessionTimeout: 20
+          inlineOperation: |
+              [
+                {
+                    "keyCode": "CP-230012",
+                    "operationSetCode": "SigntoolSign",
+                    "parameters": [
+                    {
+                        "parameterName": "OpusName",
+                        "parameterValue": "Microsoft"
+                    },
+                    {
+                        "parameterName": "OpusInfo",
+                        "parameterValue": "http://www.microsoft.com"
+                    },
+                    {
+                        "parameterName": "Append",
+                        "parameterValue": "/as"
+                    },
+                    {
+                        "parameterName": "FileDigest",
+                        "parameterValue": "/fd \"SHA256\""
+                    },
+                    {
+                        "parameterName": "PageHash",
+                        "parameterValue": "/NPH"
+                    },
+                    {
+                        "parameterName": "TimeStamp",
+                        "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                },
+                {
+                    "keyCode": "CP-230012",
+                    "operationSetCode": "SigntoolVerify",
+                    "parameters": [
+                    {
+                        "parameterName": "VerifyAll",
+                        "parameterValue": "/all"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                }
+              ]
+          signConfigType: inlineSignParams
+      - template: ../templates/install-dotnet3.yaml
+      - task: DotNetCoreCLI@2
+        displayName: "Functions Binding nuget package"
+        inputs:
+          buildProperties: OutDir=$(Build.BinariesDirectory)/publish/Microsoft.Azure.WebJobs.Extensions.EdgeHub
+          command: pack
+          nobuild: true
+          packDirectory: $(Build.BinariesDirectory)/publish/
+          packagesToPack: "**/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj"
+          versionEnvVar: version
+          versioningScheme: byEnvVar
+      - task: DotNetCoreInstaller@0
+        displayName: "Install .NET Core 2.1 - Needed for code sign on linux"
+        inputs:
+          version: "2.1.700"
```

Comment:
Install dotnet 3 for nuget package, then install dotnet 2.1 for the sign. There should be a way to have multiple dotnet versions on the agent. I wasn't able to get it to work with both installed from the beginning.